.github/workflows/build_and_push_image_on_tag.yml

```diff
@@ -33,7 +33,6 @@ jobs:
           git checkout $latestTag
           echo "Latest tag is $latestTag"
           podman --version
-          cd Dockerfiles/midstream/
-          podman build -t midstream_image:$latestTag -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=$latestTag
+          podman build -t midstream_image:$latestTag -f Dockerfiles/midstream/Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=$latestTag
```

Comment:
The tag `OPERATOR_TEST_PLAYBOOKS_TAG` is not used in any dockerfile

We should rely on default value in Dockerfile for `OPERATOR_SDK_VERSION`
Otherwise people will need to update the same number on more places.
And people tend to forget :-) We should just remove it from github actions

Reply:
@samvarankashyap bump